client/components/credit-card-form-fields/index.jsx

```diff
@@ -135,9 +136,27 @@ export class CreditCardFormFields extends React.Component {
 
 					{ this.createField( 'cvv', Input, {
 						inputMode: 'numeric',
-						label: translate( 'CVV', {
-							context: '3 digit security number on credit card form',
-						} ),
+						label: translate(
+							'CVV {{infoPopover}}{{image/}}This is the 3-digit number printed on the ' +
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?